tasks/ss-main.yml

```diff
@@ -207,9 +197,56 @@
 
 
 ###########################################################
-#   5- Database config - moved to separate files
+#   5- SS worker
+###########################################################
+
+- name: "Check if django-celery is in the pip requirements"
+  command: "grep '^django-celery' base.txt"
+  args:
+    chdir: "{{ archivematica_src_dir }}/archivematica-storage-service/requirements"
+  register: "grep_celery"
+  tags: "amsrc-ss-worker"
+
+- set_fact:
+    ss_worker: true
+  when: "grep_celery.stdout!=''"
+  tags: "amsrc-ss-worker"
+
+- name: "Install redis-server package if required"
+  apt:
+    pkg: "redis-server"
+    state: "latest"
+  when: "ss_worker|bool"
+  tags: "amsrc-ss-worker"
+
+- name: "Enable and start redis-server if not already running"
+  service:
+    name: "redis-server"
+    state: "started"
+    enabled: "yes"
+  when: "ss_worker|bool"
+  tags: "amsrc-ss-worker"
+
+- name: "Template celery upstart file if required"
+  template:
+    src: "etc_init_archivematica-storage-service-celery-worker.conf.j2"
+    dest: "/etc/init/archivematica-storage-service-celery-worker.conf"
```

Comment:
Should we name it as `archivematica-storage-service-worker`? Or you think that the `celery` part is important?